scripts/tls-common.sh

```diff
@@ -16,9 +20,18 @@ export IPXE_KEY_FILE=/certs/ipxe/tls.key
 
 export RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-"false"}
 
+# By default every cert has to be signed with Ironic's
+# CA otherwise node image and IPA verification would fail
 export MARIADB_CACERT_FILE=/certs/ca/mariadb/tls.crt
+export IRONIC_CACERT_FILE=/certs/ca/ironic/tls.crt
 
 export IPXE_TLS_PORT="${IPXE_TLS_PORT:-8084}"
+export IMAGE_SERVER_TLS_PORT="${HTTPD_TLS_PORT:-8085}"
```

Comment:
Regardless of the port discussion above, let's avoid 80XX ports. We already have a known conflict between Ironic's RPC at 8089 and k-nmstate. We already use 6180 and 6183, maybe pick something relatively free under 618X?

Reply:
done